core/main.tf

```
@@ -0,0 +1,9 @@
+terraform {
+  cloud {
+    organization = "devops-ic"
+
+    workspaces {
+      name = "staging"
```

sorry, missed that in last review, why it is `staging`? it should be both `production` and `staging` and to achieve that you need to create these two workspaces and use same tag for them, after that you can configure it as per suggestion
```suggestion
      tags = ["aws-infrastructure"]
```

---

Thanks for your suggestion, I'm struggling when setting up the `production` environment.
I updated it in bec8574 ✅